packages/suite/src/components/wallet/WalletLayout/AccountBanners/TaprootBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import { Translation } from 'src/components/suite';
+import { CloseableBanner } from './CloseableBanner';
+import { useSelector } from 'src/hooks/suite/useSelector';
+import { useDispatch } from 'src/hooks/suite/useDispatch';
+import { setFlag } from 'src/actions/suite/suiteActions';
+import { getBip43Type } from '@suite-common/wallet-utils';
+import { Account } from 'src/types/wallet';
+import { BannerPoints } from './BannerPoints';
+
+interface TaprootBannerProps {
+    account?: Account;
+}
+
+export const TaprootBanner = ({ account }: TaprootBannerProps) => {
+    const { taprootBannerClosed } = useSelector(state => state.suite.flags);
+    const dispatch = useDispatch();
+
+    const isVisible = !taprootBannerClosed && account && getBip43Type(account.path) === 'bip86';
```

Comment:
The `TaprootBanner` component was rendered in `AccountEmpty` component. `AccountEmpty` component is rendered only when condition `account.empty` is `true`. May be `account.empty` should be here as well. What do you think? @jvaclavik

Reply:
Sounds like it's not necessary but up to you guys 🤷‍♂️

Reply:
Asked @MiroslavProchazka. Let's wait a bit for his answer.

Reply:
Fixed here 1a965bba036f2e4f598b9c7f9b70444a3571e379